kotlin-react/src/main/kotlin/react/useCallback.kt

```diff
@@ -10,4 +10,17 @@ inline fun <T : Function<*>> useCallback(
     vararg dependencies: dynamic,
     callback: T,
 ): T =
-    useCallback(callback, dependencies)
+    rawUseCallback(callback, dependencies)
+
+/**
+ * Only works inside [functionalComponent]
+ * @see <a href="https://reactjs.org/docs/hooks-state.html#hooks-and-function-components">Hooks and Function Components</a>
+ */
+@Deprecated(
+    message = "Inconsistent hooks API",
+)
+inline fun <T : Function<*>> useCallback(
+    callback: T,
```

Comment:
I have added inline similarly with other code, I think that it is ok that these hooks are inlined, because there isn't any additional runtime logic here, like in effect hooks, so compiled code should look like code on clean JS without any generated Kotlin code

Reply:
`noinline` - modifier for `callback` parameter
Other function parameters have it

Reply:
The compiler complains about `T` not being a function parameter.